projects/core/src/occ/services/occ-endpoints.service.ts

```diff
@@ -110,7 +110,7 @@ export class OccEndpointsService {
       const { urlParams, queryParams } = attributes;
 
       if (urlParams) {
-        url = StringTemplate.resolve(url, attributes.urlParams, true);
+        url = StringTemplate.resolve(url, urlParams, true);
```

Comment:
I guess you don't need to destructure urlParams and can use `attributes.urlParams` everywhere instead.

Reply:
yes, but destructure it make the codes more clean.